src/libraries/System.Private.CoreLib/src/System/IO/PersistedFiles.Browser.cs

```diff
@@ -0,0 +1,10 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.IO
+{
+    internal static partial class PersistedFiles
+    {
+        internal static string GetHomeDirectory() => "/";
```

Comment:
this will make us inconsistent with the value of the `HOME` env variable that emscripten sets which will be confusing to other parts of the stack.

E.g. both of these return `/home/web_user` right now:
```
Environment.GetEnvironmentVariable("HOME")
Environment.GetFolderPath(Environment.SpecialFolder.UserProfile)
```

Reply:
> will be confusing to other parts of the stack

Could you clarify what do you mean by that? I'm fine to tweak the path to something else but `/home/web_user` is not what we should use as it does not make much sense.

Reply:
we could return Environment.GetEnvironmentVariable("HOME") here and set "HOME" to whatever we want in driver.c if it is unset. Does UserProfile get the value from "HOME" as well?

Reply:
I'm still not convinced this is necessary. The Unix implementation has to use HOME to get unique root location, in the browser case `/` is already more than user-specific and the implementation created subfolders starting from there.

Reply:
Well at the moment it is causing System.Tests.EnvironmentTests.GetFolderPath_Unix_PersonalIsHomeAndUserProfile to fail.  I think it makes sense to set to something other than / just to have a namespace maybe "/User"?

Reply:
Tweaked it based on the feedback

Reply:
a lot of code uses `Environment.GetEnvironmentVariable("HOME")` and I don't think we should second-guess what emscripten sets up since they explicitly create this (https://emscripten.org/docs/api_reference/Filesystem-API.html).

The way you have it right now would mean we'd need to create `/Users/Browser` in the library_mono.js or you wouldn't be able to write into the dir. I think it'd be more consistent to just return the value of the `HOME` env var.